homeassistant/components/tplink/sensor.py

```diff
@@ -115,6 +120,66 @@ class TPLinkSensorEntityDescription(
     TPLinkSensorEntityDescription(
         key="alarm_source",
     ),
+    TPLinkSensorEntityDescription(
+        key="main_brush_remaining",
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.HOURS,
```

Comment:
The native unit should be set to seconds. and set `suggested_unit_of_measurement` to Hours